poker/src/utils/updateStatistics.ts

```diff
@@ -0,0 +1,42 @@
+import { MAX_PERCENT } from '../constants';
+import { IssueChatItem, StatisticCard } from '../types/common';
+
+interface TempType {
+  value: string;
+  count: number;
+}
+
+const createCardValueAndCountOfVotesArray = (array: Array<IssueChatItem>): Array<TempType> => array
+.reduce((res: Array<TempType>, item) => {
+  if (item.value !== '-') {
+    const index = res.findIndex((el) => el.value === item.value);
+    if (index !== -1) res[index] = { value: item.value, count: res[index].count + 1 };
+    else res.push({ value: item.value, count: 1 });
+  }
+
+  return res;
+}, []);
```

Comment:
hard to read and understand

Reply:
If there is enough time before the deadline, I will refactor the implementation of updateStatistics function.